CryptoTechReminderSystem/Gateway/IDeveloperRetriever.cs

```diff
@@ -5,6 +5,6 @@ namespace CryptoTechReminderSystem.Gateway
 {
     public interface IDeveloperRetriever
```

Comment:
This interface needs to be renamed to `ISlackDeveloperRetriever` as we have done with `IHarvestDeveloperRetriever`.

Reply:
File name will also need to match the new interface name.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/9/commits/005bbf02ace771db1e79f7f6f9c3d1a71818dcf1